src/core/sys/posix/pthread.d

```diff
@@ -261,14 +261,70 @@ else
     static assert(false, "Unsupported platform");
 }
 
-int pthread_atfork(void function(), void function(), void function());
-int pthread_attr_destroy(pthread_attr_t*);
-int pthread_attr_getdetachstate(in pthread_attr_t*, int*);
-int pthread_attr_getschedparam(in pthread_attr_t*, sched_param*);
-int pthread_attr_init(pthread_attr_t*);
-int pthread_attr_setdetachstate(pthread_attr_t*, int);
-int pthread_attr_setschedparam(in pthread_attr_t*, sched_param*);
-int pthread_cancel(pthread_t);
+version( CRuntime_Glibc )
```

Comment:
No, [glibc is the GNU C library](https://en.wikipedia.org/wiki/GNU_C_Library) and is only used on desktop/server linux.  The BSDs have their own libc variants, as do the other UNIX spinoffs.